.github/workflows/E2E.yml

```diff
@@ -0,0 +1,44 @@
+name: E2E
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+jobs:
+  e2e:
+    timeout-minutes: 15
+    runs-on: ubuntu-22.04
+
+    steps:
+      - uses: actions/checkout@v3.3.0
+      - name: Install prerequisites
+        run: sudo apt-get update && sudo apt-get install -y jq
```

Comment:
`jq` is [already bundled in ubuntu-22.04](https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md)

Reply:
Thanks, good to know - removed the install.